File: ERP.Models/Domain/Mill.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace ERP.Domain.Models
{
    public class Mill
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Product { get; set; }

        public string ManagerName { get; set; }
        public string ManagerCell { get; set; }

        [ForeignKey("City")]
        public int CityId { get; set; }
        public City City { get; set; }

        [ForeignKey("Province")]
        public int ProvinceId { get; set; }
        public Province Province { get; set; }

        public ICollection<MillFabric> MillFabrics { get; set; }

    }
}

File: ERP/Client/src/app/mill/mill.ts
//import { Province } from './mill';

export class mill {
    Id: number;
    Name: string;
    Product: string;
    ManagerName: string;
    ManagerCell: number;
    CityId: number;
    ProvinceId: number;
    FabricId: number;
   // province: Province;

    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}